config/routes.rb

```diff
@@ -1,19 +1,32 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
+  get 'sessions/create'
+
   # devise_for :users
```

Comment:
- [ Required ] Good job deleting some of the commented code lines from your project as requested by the previous reviewer, however, this file still has commented code. Please consider deleting the commented pieces of code from your codebase to make your code clean and professional.